reference/tools/gnu/pkgconfig.rst

```diff
@@ -35,6 +38,45 @@ Use the ``pc`` file information to fill a ``cpp_info`` object:
         pkg_config.fill_cpp_info(self.cpp_info, is_system=False, system_libs=["m", "rt"])
 
 
+However, ``PkgConfig`` will invoke the ``pkg-config`` executable to extract the information from the ``.pc`` file.
+The ``pkg-config`` executable must be available in the system path for this case, otherwise, it will fail when installing the consumed package.
+
+
+Using pkg-config from Conan package instead of system
+-----------------------------------------------------
+
+.. include:: ../../../common/experimental_warning.inc
+
+In case not having ``pkg-config`` available in the system, it is possible to use the ``pkg-config`` executable provided by a Conan package:
```

Comment:
```suggestion
In case of not having ``pkg-config`` available in the system, it is possible to use the ``pkg-config`` executable provided by a Conan package:
```